doc/rst/source/events.rst

```diff
@@ -18,7 +18,7 @@ Synopsis
 [ |-C|\ *cpt* ]
 [ |-D|\ [**j**\|\ **J**]\ *dx*\ [/*dy*][**+v**\ [*pen*]] ]
 [ |-E|\ **s**\|\ **t**\ [**+o**\|\ **O**\ *dt*][**+r**\ *dt*][**+p**\ *dt*][**+d**\ *dt*][**+f**\ *dt*][**+l**\ *dt*] ]
-[ |-F|\ [**+a**\ *angle*][**+f**\ *font*][**+j**\ *justify*][**+r**\ [*first*]\|\ **+z**\ [*format*]] ]
+[ |-F|\ [**+a**\ *angle*][**+f**\ *font*][**+j**\ *justify*][**+r**\ [*first*]\|\ **z**\ [*format*]] ]
```

Comment:
This one looks very confusing.

Reply:
It is trying to show that **+a**, **+f,** **+j** are all independent modifiers (so you can use 0-3 of those) while you can only use either **+r** or **+z** but not both.  Both of those two take an optional argument (_first_ and _format_).  I think it is correctly typeset.

Reply:
OK.